cmd/api/src/api/saml/saml.go

```diff
@@ -266,29 +269,71 @@ func domainValue(host url.URL) string {
 	}
 }
 
+func (s ProviderResource) auditSAMLLogin(requestContext context.Context, user model.User, status model.AuditLogEntryStatus, err error) error {
+	if commitID, idErr := uuid.NewV4(); idErr != nil {
```

Comment:
I think this makes sense in the current context since we can tie a single request to the request_id, but since the calls to `auditSAMLLogin` all happen within the same endpoint, I'm thinking we may want to pass in the `commitID` rather than generate it each call

Reply:
Resolved via reusing [auditLogin](url)